src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -23,20 +23,12 @@ const defaultItems: ItemType[] = [
 ];
 
 function SelectExample({
-  disabled,
   textOnly,
   items = defaultItems,
   ...rest
 }: Pick<
   SelectNextProps<ItemType>,
-  | 'aria-label'
-  | 'aria-labelledby'
-  | 'buttonClasses'
-  | 'containerClasses'
-  | 'listboxClasses'
-  | 'disabled'
-  | 'right'
-  | 'listboxAsPopover'
```

Comment:
This local `SelectExample` component is still used in some cases, but I have removed all props which are no longer needed because they have been moved to example files.